frontend/src/components/Button/Button.scss

```diff
@@ -49,24 +56,38 @@
 }
 
 .secondaryButton {
-  padding: (8 * $px) (16 * $px);
-  color: #303030;
-  background-color: #f2f2f2;
-  border-color: #c4c4c4;
-  border-radius: 8 * $px;
+  @include btn-typography();
```

Comment:
because it looks ugly, not going to fix. it's from the category: fix/discuss with the designers when time comes

Reply:
yes, but unlike you I don't treat this like: I am told to do and I have to do, I treat the project like I have option to negotiate with designers the design, and say: look these margins seem to wide, there it seems weird, and so on and so forth. So it's constant negotiation. And this is just minor one which I don't wan to spend time on right now because it's unessential for now.

Reply:
And I like this margin and because of this you need to ask designers first and do not use your opinion as a good point to do what you want (but ok)

Reply:
No, you understand this's totally wrongly and in totally submissive way :putya:

In this position I leave this decision up to me: what to discuss and what to not. Actually I don't discuss anything, just point in my mind to ask someday/on next meeting. Because thankfully I am not you and I understand that if I asked about any such small detail, the development of MVP would take much more time, but underlying it's one more time: for MVP I don't make the goal to correspond totally. It would be ideally, but I am sure we won't get in time anyway, though, it's not the reason to make it if we have time.

Why I do so? Because one day designers will look over the design and point to all such things. If they don't, it would mean it's so insignificant that they don't even notice the difference, why then I should to spend time now?

If you perceive the mock up design as 100% goal to reach, I treat it as good to reach but 
1. on this stage it's ok to miss something
2. it's up to me to discuss with designers some weird things, and make sure they really insist on doing something.

I including sometimes I just don't do it in a wrong way because it's easier and because it seems to be as better/not essential to follow for now but will be discussed later

Reply:
It's a way of treating: you treat it's like ok, I got the task I have to do. But I treat it like: ok I think the task should be this because of this reason and this, so I build the reality myself :putya:

Reply:
I am sure you didn't even run the mock in presentation mode to look how it will look on your screen :putya: Because in the mock up, it looks: oh nice design, nice margin, but if you run this in presentation mode, you will see that 15px might be pretty big space.

Reply:
Last example of this: I made horizontal lines (borders) in the table shorter, no as wide as screen size and told to Darya about my concerns today. And you know what? She agreed that it was perhaps not the best decision. and such things, when we discuss something happens constantly under the hood, but you still seem to think the mock up is high quality undeniable decision to follow :putya: